### PR TITLE
Alkwa/fix warnings

### PR DESCRIPTION
**Pull request by alkwa-msft**
## Purpose
There were a few warnings popping up in the console that we could solve so we did.

Properly using ref variables for video stream components.
Fixing some linter issues.
Removing the remove participant function since we didn't connect it in properly and it has never worked.

## Does this introduce a breaking change?

```
[ ] Yes
[✔ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[✔ ] Code style update (formatting, local variables)
[ ✔] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Ran it

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by alcail — COMMENTED** [review]
Still need to fix the "GitHub Actions" ones, but otherwise sounds good :)

**Review by JamesBurnside — APPROVED** [review]
Just a couple more that github actions have left remarks on, otherwise LGTM!

File: Calling/ClientApp/package.json [hunk 1]
```diff
@@ -65,8 +65,9 @@
     "eject": "react-scripts eject",
     "prettier": "prettier --write --config .prettierrc **/*.js **/*.ts",
     "prettier:check": "prettier --check --config .prettierrc **/*.js **/*.ts",
```
Comment (JamesBurnside) [inline Calling/ClientApp/package.json:67]: These prettier commands need to check tsx and jsx files also

**Comment by alkwa-msft** [comment]
In this PR, there was a leaked secret. The secret has since been rotated

**Pull request merged**